### DISCUSSION TITLE
uv install in yaml env file

### DISCUSSION THREAD

**Discussion by pyinstallerbaggins** [conda-forge/miniforge#642] [Q&A]
Can anyone advise on how to implement uv for miniforge?

# environment.yaml
name: env
channels:
  - conda-forge
dependencies:
  - python=3.10
  - pip=24.0
  - uv=0.4.9
  - pip:
    - uv install -r requirements.txt
![image](https://github.com/user-attachments/assets/e77b2174-30a3-4ac3-8683-12df4cea5a41)

**Comment by hmaarrfk** [comment]
uv = pip replacement.

it should work with what you provided.

Please provide the requirements.txt file and the FULL output (not partially clipped) of the terminal.
